coverage-tests.js

```diff
@@ -2029,7 +2029,7 @@ test('should return aborted tests in getAllTestResults', {
     eyes.open({appName: 'Applitools Eyes SDK'})
     eyes.check()
     const abortResult = eyes.abort()
-    assert.equal(abortResult.getIsAborted(), true)
+    assert.equal(abortResult.isAborted, true)
```

Comment:
Oh, so in Java it's `.isAborted` and `setAborted`? In JS it's `.getIsAborted` and `.setIsAborted`.